exporters/prometheus/exporter_test.go

```diff
@@ -431,6 +432,47 @@ func TestPrometheusExporter(t *testing.T) {
 				counter.Add(ctx, 5, otelmetric.WithAttributeSet(attrs2))
 			},
 		},
+		{
+			name:         "monotonic sum with exemplars",
+			expectedFile: "testdata/monotonic_sum_with_exemplars.txt",
+			recordMetrics: func(ctx context.Context, meter otelmetric.Meter) {
+				tp := sdktrace.NewTracerProvider(
+					sdktrace.WithSampler(sdktrace.AlwaysSample()),
+					sdktrace.WithResource(resource.NewWithAttributes(
+						semconv.SchemaURL,
+						semconv.ServiceNameKey.String("example"),
+					)),
+				)
+				tracer := tp.Tracer("example")
+				ctx, span := tracer.Start(ctx, "example")
+				defer span.End()
+
```

Comment:
Why do we need traces in a metrics exporter?

```suggestion
```

Reply:
I assume you did this to generate a sampled trace context.  You can do that without using the trace SDK, similar to https://github.com/open-telemetry/opentelemetry-go/blob/7a153a01c6f75f05b63909bd08d823de65939bd2/exporters/prometheus/exporter_test.go#L976

Reply:
Yes, exactly.  I found this bug using a middleware for gin that shares the context from gin for both traces and metrics reporting.  Agree it doesn't make sense normally attach trace information to a metric context.